src/utils/flowtype.js

```diff
@@ -64,6 +64,53 @@ export type FileDescriptor = {
 	md5sum: string;
 };
 
+// Semantic types
+export type SemanticId = {
+	timestamp: number;
+	date: number;
+};
+export type SemanticContext = {
+	"@language": string;
+	"gml": string;
+	"iwfs": string;
+	"xlink": string;
+	"xsd": string;
+	"qudt": string;
+	"unit": string;
+	"openvocab": string;
+
+}
+
+export type SemanticColumn = {
+	name: string;
+	titles: string;
```

Comment:
maybe use `Title`? so it would be more consistent with other headers

Reply:
and maybe should do align left?

Reply:
![image](https://github.com/IN-CORE/incore-ui/assets/63822845/f587fc95-eec0-4630-aa71-f778c156beff)

Reply:
Hmm it is aligned for me

Reply:
Anyways Chen mentioned we will remove the file after this release